sim/battle-stream.ts

```diff
@@ -117,11 +117,7 @@ export class BattleStream extends Streams.ObjectReadWriteStream {
 			break;
 		case 'eval':
 			/* tslint:disable:no-eval */
-			let battle = this.battle!;
-			let p1 = battle && battle.p1;
-			let p2 = battle && battle.p2;
-			let p1active = p1 && p1.active[0];
-			let p2active = p2 && p2.active[0];
+			const battle = this.battle!;
```

Comment:
These are all used; it's an eval function...

Reply:
Specifically, this eval function is for a REPL. This is actually the one place we're supposed to have unused variables.

Reply:
Done